juristische-zitierweise-schweizer-stil.csl

```diff
@@ -0,0 +1,388 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" default-locale="de-DE">
```

Comment:
`default-locale="de-DE"`

Note that we have a dedicated CSL locale style for Swiss: https://github.com/citation-style-language/locales/blob/master/locales-de-CH.xml (locale code "de-CH")